wrapper.html

```diff
@@ -135,26 +136,29 @@
 	<meta name="twitter:site:id" content="{{ templateset.custom_fields.site_url }}" />
 
 	{% block meta_additions %}{% endblock %}
-<script src="https://www.google.com/recaptcha/api.js" async defer></script>
-	<!-- Mapbox GL Source -->
-	<script src='https://api.mapbox.com/mapbox-gl-js/v1.7.0/mapbox-gl.js'></script>
-	<link href='https://api.mapbox.com/mapbox-gl-js/v1.7.0/mapbox-gl.css' rel='stylesheet' />
-	<!-- End -->
-
-	<!--JQuery UI CSS-->
-	<link href='https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.css' rel='stylesheet' />
-	<!--End-->
-
-	<!--TimePicker CSS -->
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/timepicker/1.3.5/jquery.timepicker.min.css"/>
-	<!--End-->
-
-
-
-
+	<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+	{% if page.type != "Donation" %}

```

Comment:
Kindly explain the thinking here. We need Mapbox outside donation pages for event pages as well. What considerations did you make?

I also believe other pages require the jquery UI import.

Reply:
Since it appears that we're not using the MapBox, JQuery UI and Timepicker libraries for the Donation pages, we thought it would be best for Pagespeed to not include them on these pages.

Do we need any of these libraries on donation pages? I can fix, if so!